src/modules/meta_producto/dto/meta_producto.dto.ts

```
@@ -4,13 +4,13 @@ export interface MetaProducto {
     area_id: number;
     ods_id: number;
     enfoque_poblacional_id: number;
-    codigo: string;
+    // codigo: string;
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)